src/components/Modals/OnboardingReview.vue

```diff
@@ -0,0 +1,514 @@
+<template>
+  <div class="onboarding">
+    <div class="onboarding-section">
+      <div class="onboarding-subHeader">
+        <span class="onboarding-subHeader--font"> Basic Information</span>
+        <span>
+          <button class="onboarding-button-previous" @click="setPage(1)">
+            <img src="@/assets/images/edit-review.svg" />
+          </button>
+        </span>
+      </div>
+      <div class="onboarding-subsection onboarding-inputs--review">
+        <div class="onboarding-subHeader2-fillRow">
+          <span class="onboarding-subHeader2-review"> Your Name</span>
+        </div>
+        <div class="onboarding-selectWrapperRow-review">
+          <div class="onboarding-inputWrapper onboarding-inputWrapper--name">
+            <label class="onboarding-label"><span> First Name </span></label>
+            <label class="onboarding-label--review"
+              ><span> {{ firstName }}</span></label
+            >
+          </div>
+          <div class="onboarding-inputWrapper onboarding-inputWrapper--name">
+            <label class="onboarding-label"><span> Middle Name </span></label>
+            <label class="onboarding-label--review"
+              ><span> {{ middleName }}</span></label
+            >
+          </div>
+          <div class="onboarding-inputWrapper onboarding-inputWrapper--name">
+            <label class="onboarding-label"><span> Last Name </span></label>
+            <label class="onboarding-label--review"
+              ><span> {{ lastName }}</span></label
+            >
+          </div>
+        </div>
+        <div class="onboarding-subHeader2-fillRow">
+          <span class="onboarding-subHeader2-review"> Your Major</span>
+        </div>
+        <div class="onboarding-selectWrapper">
+          <div class="onboarding-selectWrapper-review">
+            <label class="onboarding-label">College*</label>
+            <label class="onboarding-label--review">{{
```

Comment:
This class has two dashes before the review, but the one right above has just one. Can this be consistent?

Reply:
Since we discussed removing the review page, can we revisit this later since it could be scrapped?